SPECS/asciidoc/asciidoc.spec

```diff
@@ -41,23 +39,24 @@ rm -rf %{buildroot}%{_infodir}
 make DESTDIR=%{buildroot} install
 
 %check
-python tests/testasciidoc.py update
-python tests/testasciidoc.py run
+python3 tests/testasciidoc.py update
+python3 tests/testasciidoc.py run
 
 %post   -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
 
 %files
 %defattr(-,root,root)
-%license COPYING
+%license COPYRIGHT
 %{_bindir}/*
 %{_sysconfdir}/*
 %{_mandir}/*
 
 %changelog
-* Sat May 09 00:21:18 PST 2020 Nick Samson <nisamson@microsoft.com> - 8.6.10-4
-- Added %%license line automatically
-
+*   Wed May 05 2021 Nick Samson <nisamson@microsoft.com> - 9.1.0-1
+-   Updated to 9.1.0, removed python2 support
```

Comment:
Could you also verify the license while you're here? Especially since it looks like the license file has at least been renamed.